### DISCUSSION TITLE
auth server requires a valid license file to start

### DISCUSSION THREAD

**Discussion by jdmedeiros** [gravitational/teleport#5991] [Q&A] [answered]
Trying out Teleport... using Terraform to create the server as [detailed here](https://github.com/gravitational/teleport/tree/master/examples/aws/terraform/starter-cluster), but it's asking for a license file. Am I trying to implement something that requires a license?

```
Mar 14 15:29:26 teleport teleport: INFO [AUTH:1]    Starting Auth service with PROXY protocol support. service/service.go:1226
Mar 14 15:29:26 teleport teleport: #033[31mERROR: #033[0minitialization failed
Mar 14 15:29:26 teleport teleport: auth server requires a valid license file to start, please set the correct license_file path under auth_service section in your teleport config or put the license into the default search location at /var/lib/teleport/license.pem
Mar 14 15:29:26 teleport systemd: teleport.service: main process exited, code=exited, status=1/FAILURE
Mar 14 15:29:26 teleport systemd: Unit teleport.service entered failed state.
Mar 14 15:29:26 teleport systemd: teleport.service failed.

```

**Comment by stevenGravy** [comment]
@jdmedeiros  you can use  a community version OSS AMI.  That does not require a license.  If there are features of Teleport [enterprise](https://goteleport.com/teleport/docs/enterprise/introduction/)  you need please [contact](https://goteleport.com/contact-us/) us.

**Comment by webvictim** [comment, marked as answer]
As @stevenGravy says, you can use an OSS AMI rather than Enterprise: `TF_VAR_ami_name ?="gravitational-teleport-ami-oss-6.0.1"`

**Reply by jdmedeiros** [reply]
Thanks, I appreciate that. I was using gravitational-teleport-ami-ent-6.0.1.
